deploy/examples/example-gke-pv.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: siddhi-pv
+  labels:
+    type: local
+spec:
+  capacity:
+    storage: 1Gi
+  storageClassName: standard
+  accessModes:
+    - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Recycle
+  nfs:
```

Comment:
Shall we merge the two PV examples and comment out the nfs and add a description saying why we need to use a remote file system. Have the minikube version by default